eunkyung/routes/postRouter.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express')
+const postController = require('../controllers/postController')
+
+const router = express.Router()
```

Comment:
module을 import하는 부분도 수정해주면 좋을 것 같습니다!
```suggestion
const express = require('express')

const postController = require('../controllers/postController')
const router = express.Router()
```